include/hepa-uv/core/hepa_task.hpp

```diff
@@ -50,7 +50,7 @@ class HepaMessageHandler {
             hepa_push_button = !hepa_push_button;
             // handle state changes here
             if (hepa_push_button) {
-                hepa_hardware.set_hepa_fan_speed(50);
+                hepa_hardware.set_hepa_fan_speed(75);
```

Comment:
don't do this in the same PR as all the other stuff please. it's completely unrelated and such a small change that it would be easy to miss browsing diffs.